andela/travel_tool_back
# #161340967 Add new Andela centers

#### What does this PR do?
This PR adds an endpoint to create new Andela centers

#### Description of Task to be completed?
- add test scripts for the new endpoint
- create an endpoint to add new centers

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
- Change into the project directory -` cd travel_tool_back`
- Pull and checkout into this branch - `ft-add-new-andela-center-161340967`
- Install project dependencies using `yarn install`
- Run database migrations using `yarn run db:rollmigrate`
- Check `.env.example` and set up your `.env` appropriately
- Start the app using make start
- Navigate to [travela](http://travela-staging.andela.com/) in your browser and login
- Open the developer tools, locate applications tab, copy `jwt-token` in the cookies
- Get back to your command line, Install dependencies and start the server - `make start`
- In`Postman`, add a `Header`, with a key `Authorization`,  and the value of the `token`.
- upgrate your role in the database to Super Administrator
- In Postman, in the `x-www-form-urlencoded` body options;
    - Create a new center
    with body such as `{ newLocation: 'Cairo, Egypt' }`

- You should be able to add a new centre at  `POST: localhost:5000/api/v1/centers`
- The response should include a message and a success flag

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#161340967](https://www.pivotaltracker.com/story/show/161340967)

#### Screenshots (if appropriate)

<img width="806" alt="screen shot 2018-10-26 at 3 06 28 pm" src="https://user-images.githubusercontent.com/32720508/47573407-24481d00-d935-11e8-8fe3-df59162116cc.png">

#### Questions:
N/A
